pkg/translator/prometheusremotewrite/helper.go

```diff
@@ -575,6 +604,44 @@ func addResourceTargetInfo(resource pcommon.Resource, settings Settings, timesta
 	addSample(tsMap, sample, labels, infoType)
 }
 
+func addScopeTargetInfo(
```

Comment:
nit: scope info and target info are different concepts.  I'd prefer naming this addScopeInfo()